### DISCUSSION TITLE
[epic] How to have funded a covid vaccine for everyone (by now)

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/diginomics#46] [Ideas]
A vaccine for COVID that is available and affordable for everyone is a perfect example of failure of existing innovation and access system for medicines and how iMED / Remuneration Rights model would have been better.

- What happened with COVID: when were vaccines created, what was cost of access
  - general lack of transparency
- What the problem was and is
- How that relates to current model for medical innovation and delivery
- How iMED model would have worked

**Comment by theo-cox** [comment]
I would say this is priority no.2 after web3 stuff given potential for coverage in mainstream news (and even something perhaps in parallel if we have a few spare hours, unlikely as that is) - doesn't take much original thought just back of the envelope calculations plugged into existing OpenRev/iMed models and packaged into a decent enough sounding article...
